addons/web/static/src/views/relational_model/dynamic_list.js

```diff
@@ -203,4 +207,51 @@ export class DynamicList extends DataPoint {
             return this._load(0, this.limit, this.orderBy);
         }
     }
+
+    async _multiSave(record) {
+        const changes = record._getChanges();
+        if (!Object.keys(changes).length) {
+            return;
+        }
+        const { _onWillSaveMultiRecords, _onSavedMultiRecords } = this.model;
```

Comment:
why this? wouldnt it be simpler to just to `this.model._onWillSaveMultiRecords(...)` when needed?